docs/source/examples/md/gru4rec_examples.md

```diff
@@ -1,59 +1,105 @@
-# Gru4Rec for session-based recommendation
+# Example to train gru4rec model with FedAvg Strategy
 
+This doc introduce how to use PaddleFL to train model with Fl Strategy.
 
-[Graph Convolutional Network \(GCN\)](https://arxiv.org/abs/1609.02907) is a powerful neural network designed for machine learning on graphs. Based on PGL, we reproduce GCN algorithms and reach the same level of indicators as the paper in citation network benchmarks.
-
-### Simple example to build GCN
+### Dependencies
+- paddlepaddle>=1.6
 
-To build a gcn layer, one can use our pre-defined ```pgl.layers.gcn``` or just write a gcn layer with message passing interface.
-```python
-import paddle.fluid as fluid
-def gcn_layer(graph_wrapper, node_feature, hidden_size, act):
-    def send_func(src_feat, dst_feat, edge_feat):
-        return src_feat["h"]
-    
-    def recv_func(msg):
-        return fluid.layers.sequence_pool(msg, "sum")
-    
-    message = graph_wrapper.send(send_func, nfeat_list=[("h", node_feature)])
-    output = graph_wrapper.recv(recv_func, message)
-    output = fluid.layers.fc(output, size=hidden_size, act=act)
-    return output
+### How to install PaddleFL
+please use the python which has installed paddlepaddle.
+```sh
+python setup.py install
 ```
 
-### Datasets
+### Model
+[Gru4rec](https://arxiv.org/abs/1511.06939) is the classical session-based recommendation model. The details implement by paddlepaddle is [here](https://github.com/PaddlePaddle/models/tree/develop/PaddleRec/gru4rec).
 
-The datasets contain three citation networks: CORA, PUBMED, CITESEER. The details for these three datasets can be found in the [paper](https://arxiv.org/abs/1609.02907).
 
-### Dependencies
+### Datasets
+Public Dataset [Rsc15](https://2015.recsyschallenge.com) 
 
-- paddlepaddle>=1.4 (The speed can be faster in 1.5.)
-- pgl
+```sh
+#download data
+cd example/gru4rec_demo
+sh download.sh
+```
 
-### Performance
+### How to work in PaddleFL
+PaddleFL has two period , CompileTime and RunTime. In CompileTime, define a federated learning task by fl_master. In RunTime, train a federated learning job by fl_server and fl_trainer .
```

Comment:
define a federated learning task by fl_master -> 
a federated learning task is defined by fl_master

train a federated learning job by fl_server and fl_trainer ->
a federated learning job is executed on fl_server and fl_trainer in distributed cluster.